### PR TITLE
[RFC] Initial walkthrough content for VS Code extension

### PR DESCRIPTION
**Pull request by rossdanlm**
[RFC] Initial walkthrough content for VS Code extension

Following the guide from VSCode tutorials here: https://code.visualstudio.com/api/references/contribution-points#contributes.walkthroughs

This is now just extremely barebones, want to get some comments on what other commands we feel should be included.

Also note that I can also add in images and markdown to the right side of panel for each item that's selected

Let's just get this out really quickly so that Tanya + Zak can help out with content + graphics in the markdown files

## Test Plan

https://github.com/lastmile-ai/aiconfig/assets/151060367/5827b98c-ac1c-4454-a337-ce9eb3c44b1a

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1273).
* #1276
* __->__ #1273

### CODE REVIEW

File: vscode-extension/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "Other"
   ],
   "main": "./out/extension.js",
+  "activationEvents": ["*"],
```
Comment (rholinshead) [inline vscode-extension/package.json:18]: Why is this needed? Also, is there a reason for `*` vs `onStartupFinished`?

Comment (rossdanlm) [reply vscode-extension/package.json:18]: Oh yea this was just for testing, removing

File: vscode-extension/package.json [hunk 2]
```diff
@@ -96,7 +96,56 @@
...
+						"title": "Install dependencies",
+						"description": "This installs the Python libraries required to use the AIConfig editor and SDK",
+						"media": {
+              "markdown": "src/welcomePage.md"
+						}
+					},
+          {
+						"id": "setupEnvironmentVariables",
+						"title": "Setup Environment Variables",
+						"description": "This creates an .env file (or uses it if it already exists) to define environment varialbes needed to run AI models. Ex: Open AI models read from the `OPENAI_API_KEY` variable",
```
Comment (rholinshead) [inline vscode-extension/package.json:125]:
```suggestion
						"description": "This creates an .env file (or uses it if it already exists) to define environment varialbes needed to run AI models. Ex: OpenAI models read from the `OPENAI_API_KEY` variable",
```
nit

**Review by rholinshead — APPROVED** [review]

File: vscode-extension/package.json (hunk 2 above)
Comment (rossdanlm) [reply vscode-extension/package.json:125]: TIL Open AI is two separate words lol, just like Hugging Face
